boards/mikroe/clicker_stm32_m4/board.yml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Ian Morris
+# SPDX-License-Identifier: Apache-2.0
+
+board:
+  name: mikroe_clicker_stm32_m4
```

Comment:
I think I'd want the name to be `mikroe_stm32_m4_clicker` as this is the commercial name, but I can also see how you'd want all clicker boards to be named clicker_something. Too bad MikroE is inconsistent in naming their products :/

From a closer look, there might actually be a rationale for having both "Clicker N for..." and "XYZ Clicker" types of products so this would make the point for trying not to deviate from the commercial name https://www.mikroe.com/search?search_query=clicker

Reply:
Modified as requested, let me know what you think...